can/interface.py

```diff
@@ -71,8 +78,13 @@ class Bus(BusABC):  # pylint: disable=abstract-method
         Set to ``None`` to let it be resolved automatically from the default
         :ref:`configuration`.
 
-    :param args:
-        ``interface`` specific positional arguments.
+    :param context:
```

Comment:
Maybe call it `config_context`?

Reply:
That's exactly what i called it at first 😄 
But the existing implementation gets `context` from kwargs, so that might break user code.

Reply:
Done, i also added it to the deprecation decorator for whoever might need it.